css/index.css

```diff
@@ -1,3 +1,394 @@
+  margin-top: 20px;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  width: 70%;
+  margin-top: 35px;
+}
+
+header nav a {
+  /* margin-right: 65px; */
+  /* width: 60%; */
+  justify-content: space-between;
+  font-size: 1.4rem;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+/* cta */
+
+.cta {
+  display: flex;
+  justify-content: space-between;
+}
+
+.cta .cta-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 400px;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0 75px 0 100px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 1.7rem;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 49%;
+  /* margin-right: 34px; */
+  font-size: 1.6rem;
+}
+
+.main-content .top-content .text-content:last-child {
+  /* margin-right: 0; */
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  margin-right: 12px;
+  font-size: 1.6rem;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+  font-size: 1.6rem;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 1.4rem;
+}
+
+/* tablet view */
+
+@media (max-width: 800px) {
+  .container {
+    width: 98%;
+    margin: 0 auto;
+  }
+
+  header {
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+
+
+  header nav {
+    width: 95%;
+    display: flex;
+    margin-top: 0px;
+    /* flex-direction: column-reverse; */
+    align-items: center;
+    justify-content: space-between;
+    /* margin-left: 50px; */
+  }
+
+  header nav a {
+    margin-top: 40px;
+    /* width: 0%; */
+    
+  }
+
+  .cta {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .cta img {
+    display: none;
+  }
+
+  .main-content {
+    width: 100%;
+  }
+
+  .middle-img {
+    width: 100%;
+  }
+}
+
+/* mobile view  */
+
+@media (max-width: 500px) {
+  .container {
+    width: 100%;
+    margin: 0 auto;
+  }
+  
+  header{
+    width: 100%;
+  }
+  
+  nav {
+    flex-direction: column;
+    width: 100%;
+  }
+
```

Comment:
- Introduce max-width media queries into your project at 800px and 500px - WELL DONE!